activities/models/post.py

```diff
@@ -959,7 +959,12 @@ def to_opengraph_dict(self) -> dict:
     def to_mastodon_json(self, interactions=None):
         reply_parent = None
         if self.in_reply_to:
-            reply_parent = Post.objects.filter(object_uri=self.in_reply_to).first()
+            # Load the PK and author.id explicitly to prevent a SELECT on the entire author Identity
+            reply_parent = (
```

Comment:
How much is this actually saving? It's not going to be a ton I think, and it's making the rest of the code harder to read (and if you just want to avoid loading all the columns, use `only` rather than `values`)

Reply:
Ah, good to know. I was looking for something like `only` and only found `values`

Reply:
Fixed!